functions_lab.js

```diff
@@ -0,0 +1,418 @@
+// # Function Exercises
+//
+// ## Question 1
+//
+// Write a function that returns the largest of two numbers. If they are equal, return the second one.
+//
+// Example:
+//
+// Input: `12, 4`
+//
+// Output: `12`
+
+console.log('=======answer#1=========')
+
+let num1 = 12;
+let num2 = 4;
+function largestOfTwoNum (num1, num2) {
+  if (num1 < num2) {
+  return num2
+  } else if (num1 > num2) {
+    return num1
+  } else {
+    return num1,num2
+  }
+}
+console.log(largestOfTwoNum(num1, num2))
+
+// #########
+
+const largestOfTwoNum = (num1, num2) => {
+  if (num1 < num2) {
+    return num2
+  } else if (num1 > num2) {
+    return num1
+  } else {
+    return num1, num2
+  }
+}
+let num1 = 12;
+let num2 = 4
+console.log(largestOfTwoNum(num1, num2))
+
+// ## Question 2
+//
+// Write a function named getDogAge that takes one argument: a dog's
+// age in human years. The function returns the dog's age in dog years.
+//The conversion rate is 1 human year to 7 dog years.
+//
+// Example:
+//
+// Input: `4`
+//
+// Output: `28`
+
+console.log('=======answer#2=========')
+
+function getDogAge (num) {
+return num * 7
+}
+let num = 4
+let result = num * 7
```

Comment:
Instead of writing let result = num * 7, you could just use the function that you just wrote. 
let result = getDogAge(num)